src/frontend/csrs-portal/src/app/app.component.ts

```diff
@@ -31,9 +33,9 @@ export class AppComponent implements OnInit {
 
   ngOnInit() {
     if (isDevMode()) {
-      console.log('Development!');
+      this.logger.log('info', 'Development!')
     } else {
-      console.log('Production!');
+      this.logger.log('info', 'Production!')
```

Comment:
See cleaner way of managing and suppressing log level
https://www.codemag.com/article/1711021/Logging-in-Angular-Applications
https://github.com/bcgov/dts-vc-issuer-service/blob/dev/angular-frontend/src/app/core/services/logger.service.ts

Reply:
I don't know why somebody added  console.log('Production!'); block into app.component.ts. I deleted this block.
You provided link about Logger service. It was already developed.